scripts/convert_yaml_to_sdf.py

```diff
@@ -0,0 +1,207 @@
+                                  'geometry': {'box': {'size': shape_items_size}}},
+                             'visual name="{} visual"'.format(shape_name):
+                                 {'pose': shape_items_pose,
+                                  'geometry': {'box': {'size': shape_items_size}}}
+                             })
+            if object_color != [0, 0, 0, 1]:
+                sdf_dict['visual name="{} visual"'.format(shape_name)] = \
+                    {'material': {'ambient': str(object_color).replace("[", "").replace("]", "").replace(",", "")}}
+
+    return sdf_dict
+
+
+def main():
+    # if len(sys.argv) <= 1:
+    #     print "Please provide a file as input"
+    #     sys.exit(1)
+
+    # Create an empty dictionary in SDF format, which can later on be exported to XML after filling below.
+    sdf_dict = {}
+
+    # First determine type of model.yaml
+    # Raw input can be replaced by fancier read_option in create-model.py
+    # file_type = raw_input("File type ('world' or 'object'): ")
+    # model_name = raw_input("Model name: ")
+
+    # Temporary replacements of user input
+    file_type = "object"
+
+    with open("model.yaml", 'r') as stream:
+        try:
+            x = yaml.load(stream)
+            # print(x)
+        except yaml.YAMLError as exc:
+            print(exc)
+    print(x)
+
+    # Filling the empty dictionary
+    if file_type == "object":
+        model_name = 'model name="{}"'.format("couch")
+        # Is object, so create a (static) model type SDF dictionary (as opposed to world), with a default pose
+        sdf_dict = {model_name: {'link name="link"': {},
+                                 'pose': "0 0 0 0 0 0",
+                                 'static': 'true'}}
+        sdf_dict_model_level = sdf_dict[sdf_dict.keys()[0]]
+        sdf_dict_link_level = sdf_dict_model_level['link name="link"']
+
+        if isinstance(x, dict):
+            if 'color' in x:
+                red = x['color']['red']
+                green = x['color']['red']
+                blue = x['color']['blue']
+                sdf_dict_link_level.update(fill_shapes(x['shape'], red, green, blue))
+            else:
+                sdf_dict_link_level.update(fill_shapes(x['shape']))
+
+            for area_items in x['areas']:
+                # sdf_dict_custom_level[area_items['name']] = {}
+                # if 'offset' in area_items:
+                #     sdf_dict_custom_level[area_items['name']].update({'offset': area_items['offset']})
+                #
+                # if 'shape' in area_items:
+                #     sdf_dict_custom_level[area_items['name']].update(fill_shapes(area_items['shape']))
+
+                print area_items
+
+            # Todo: All custom parameters
+
+        elif isinstance(x, list):
+            for item in x:
+                print 'List item:', item
+    elif file_type == "world":
+        world_name = 'world name="{}"'.format("rwc2018")
```

Comment:
Again rwc2018 is hardcoded. It would be nice if this could be made into an input